### PR TITLE
Navigation Widget

### PR DESCRIPTION
**Pull request by CorentinGaut**
## Navigation UI
Creation of an UI to navigated differenty in the UD-Viz scene and have more information about your localisation.

The UI is composed of : 
- A slider zoom that move the camera on his point of view with a specific constant.
- A compass to indicate north in the view

This UI work on the [PlanarView](http://www.itowns-project.org/itowns/docs/#api/View/PlanarView) of itowns which is different to the [Navigation Widget](http://www.itowns-project.org/itowns/docs/#api/Widgets/Navigation) of this library that only work on [GlobeView](http://www.itowns-project.org/itowns/docs/#api/View/GlobeView).

An example of the widget is integrate in the AllWidget page.

### CODE REVIEW

File: src/Widgets/Extensions/Navigation/Navigation.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+import { ModuleView } from '../../Components/ModuleView/ModuleView';
+import * as THREE from 'three';
+import './navigation.css';
+
+export class Navigation extends ModuleView {
+  constructor(view3D) {
+    super();
+    
+    //Scene renderer
+    let renderer = view3D.mainLoop.gfxEngine.renderer;
```
Comment (mathieuLivebardon) [inline src/Widgets/Extensions/Navigation/Navigation.js:11]: prefer const than let when it's possible

File: src/Widgets/Extensions/Navigation/navigation.css [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+    /* Magic trick to center */
+    left: 0;
+    right: 0;
+    margin-left: auto;
+    margin-right: auto;
+    bottom: 70px;
+    z-index: 10;
+    background-color: #00000088;
+    list-style-type: none;
+
```
Comment (mathieuLivebardon) [inline src/Widgets/Extensions/Navigation/navigation.css:15]: remove this space

**Comment by clementcolin** [comment]
Dumb question from the outside, but why do not we extend the itowns widget that works only on globe view to make it work on planar view? It is meant to be integrated in the futur ?

**Comment by CorentinGaut** [comment]
@clementcolin it's a different approach to Itowns zoom but in the futur it will be an enhancement of the navigation widgets from Itowns

**Pull request closed without merging**